configurationTemplates/harmonic_worker.conf.j2

```diff
@@ -0,0 +1,9 @@
+[program:harmonic_worker]
+user=root
+directory=/home/ubuntu/HarmonicIO
+command=python3.5 -m harmonicIO.worker
+autostart=true
+autorestart=true
+startretires=3
+stdout_logfile=/home/ubuntu/HarmonicIO/logs/harmonic_worker.log
```

Comment:
use ~ ?

Reply:
(this code is not portable to other OSs as it stands, creates a hard dependency where there doesn't need to be?